packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html

```diff
@@ -331,37 +331,22 @@
                             class="oui-badge"
                             data-ng-class="{
                                     'oui-badge_success': $ctrl.server.monitored,
-                                    'oui-badge_error': !$ctrl.server.monitored
+                                    'oui-badge_warning': !$ctrl.server.monitored
                                 }"
-                            data-translate="{{'server_configuration_monitoring_' + ($ctrl.server.monitored && 'activated' || 'disabled')}}"
-                            data-oui-tooltip="{{ ($ctrl.server.monitored && 'server_configuration_monitoring_activate_information' | translate) || ('server_configuration_monitoring_deactivate_information' | translate) }}"
+                            data-ng-bind="$ctrl.getMonitoringStatusTranslation()"
```

Comment:
```suggestion
                            data-rtanslate="$ctrl.getMonitoringStatusTranslation()"
```
Method could return only the generated translation key